src/utils.js

```diff
@@ -9,19 +9,17 @@ const colorGradientLength = colorGradient.length;
 
 function getNodeBackgroundColor(value, maxValue) {
   return backgroundColorGradient[
-    Math.round(value / maxValue * (backgroundColorGradientLength - 1))
+    Math.round((value / maxValue) * (backgroundColorGradientLength - 1))
```

Comment:
This change looks manual since it's not necessary ^ Prettier should format this sort of thing 😄

Reply:
Whoops I think eslint --fix did this actually, but will remove these changes.

Reply:
Oh! My bad then 😄